tests/foreman/api/test_rhc.py

```diff
@@ -0,0 +1,122 @@
+"""Test class for RH Cloud connector - rhc
+
+:Requirement: Remoteexecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: RHCloud-CloudConnector
+
+:Team: Platform
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from wait_for import wait_for
+
+from robottelo import constants
+from robottelo.utils.issue_handlers import is_open
+
+
+@pytest.fixture(scope='module')
+def fixture_enable_rhc_repos(module_target_sat):
+    """Enable repos required for configuring RHC."""
+    # subscribe rhc satellite to cdn.
+    module_target_sat.register_to_cdn()
+    if module_target_sat.os_version.major == 8:
+        module_target_sat.enable_repo(constants.REPOS['rhel8_bos']['id'])
+        module_target_sat.enable_repo(constants.REPOS['rhel8_aps']['id'])
+    else:
+        module_target_sat.enable_repo(constants.REPOS['rhscl7']['id'])
+        module_target_sat.enable_repo(constants.REPOS['rhel7']['id'])
+
+
+@pytest.fixture(scope='module')
+def module_rhc_org(module_target_sat, module_org):
```

Comment:
Same as what @shweta83 said above for this fixture. We have almost the same fixture in the UI test. If that check for `crc_env='prod'` applies here we should just move both fixtures to a common file like `pytest_fixtures.component.rh_cloud`

Reply:
Same reason as above https://github.com/SatelliteQE/robottelo/pull/11395#discussion_r1210160371.